Cargo.toml

```diff
@@ -28,19 +28,19 @@ bench = false
 test = true
 
 [dependencies]
-num-integer = "0.1.39"
-num-traits = { version = "0.2", default-features = false }
+num-integer = { version = "0.1.39", default-features = false }
+num-traits = { version = "0.2", default-features = false , features = ["libm"] }
 num-complex = { version = "0.3", default-features = false }
 
 rayon = { version = "1.0.3", optional = true }
 
-approx = { version = "0.4", optional = true }
+approx = { version = "0.4", optional = true , default-features = false }
 
 # Use via the `blas` crate feature!
 cblas-sys = { version = "0.1.4", optional = true, default-features = false }
 blas-src = { version = "0.6.1", optional = true, default-features = false }
 
-matrixmultiply = { version = "0.2.0" }
+matrixmultiply = { version = "0.2.0", default-features = false}
```

Comment:
ndarray/std must enable matrixmultiply/std; runtime target feature detection is one of the most important features of matrixmultiply

Reply:
Is there any solution? like first detect features and then introduce modules

Reply:
just write std = ["matrixmultiply/std"] under features and it's what we need

Reply:
Ok, got it.